src/components/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useCallback, useState } from 'react';
+import PropTypes from 'prop-types';
+
+import { FlexContainer } from '../FlexContainer';
+import { AccordionProps } from './Accordion.types';
+import AccordionCollapsible from './AccordionCollapsible';
+
+const Accordion: React.FC<AccordionProps> = ({
+  isCollapsedOnOpen = true,
+  items,
+  ...props
+}) => {
+  const [openIndexes, setOpenIndexes] = useState(
+    items.filter((item) => item.isOpen).map((item, index) => index),
```

Comment:
I'm a bit wary of using indices as keys (since data can be sorted in the parent and issues happens) although there's assumption that Accordion will always get some constant data. 

I would still much rather add an `id` to the items array

Reply:
@vaclav-zeman we can do it, but I was thinking that it kinda puts the burden to the component user and he will just snap numbers there all the same. But I get your point.

Reply:
Yep, trade off for sure. But it's fine if user adds numbers since they will stick to the item even after sorting/removing item

Reply:
@vaclav-zeman done 😌